router/broker.go

```diff
@@ -98,6 +131,30 @@ func (b *broker) role() wamp.Dict {
 	return brokerRole
 }
 
+// PreInitEventHistoryTopics initializes storage for event history enabled topics.
+func (b *broker) PreInitEventHistoryTopics(evntCfgs []*TopicEventHistoryConfig) error {
+	for _, topicCfg := range evntCfgs {
+		if !topicCfg.Topic.ValidURI(b.strictURI, topicCfg.MatchPolicy) {
+			return fmt.Errorf("invalid topic/match configuration found: %s %s", topicCfg.Topic, topicCfg.MatchPolicy)
```

Comment:
If you follow my suggestion below you can reuse the wrapper `topicConfigError` and ideally return an error that describe why the URI is not valid.